Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.7-alpine
+
+RUN apk add --no-cache --virtual build-dependencies gcc musl-dev
```

Comment:
i don't think this is even needed, is it ?

Reply:
unfortunately it is to build the regex module...
```
    ...
    building '_regex' extension
    creating build/temp.linux-x86_64-3.7
    creating build/temp.linux-x86_64-3.7/regex_3
    gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/local/include/python3.7m -c regex_3/_regex.c -o build/temp.linux-x86_64-3.7/regex_3/_regex.o
    unable to execute 'gcc': No such file or directory
    error: command 'gcc' failed with exit status 1
    ...```

Reply:
mkay. guess it need to stay then :)